src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService use = new UserService();
```

Comment:
We can add field with type `UserService` to our class, so we won't need to create an instance every time
Also, consider another name for this variable (good one could be `userService`)